nixos/modules/services/networking/networkmanager.nix

```
@@ -325,16 +318,75 @@ in {
         '';
       };
 
+      enableFortiSSL = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable the FortiSSL plugin.
+          </para><para>
+          If you enable this option the <literal>networkmanager_fortisslvpn</literal>
```

```suggestion
          If you enable this option, the <literal>networkmanager_fortisslvpn</literal>
```

Same as last time: cascade this change please :)

---

Thanks :)